website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -65,6 +65,27 @@ The most common types are:
  - `Gauge` Value can be increased and decreased as needed.
 5. `_total` suffix should be used for accumulating count. If your metrics has labels with high cardinality, like `pod`/`container` it usually means that you can aggregate it more, thus it will not require `_total` suffix.
 
+#### Recommended Metrics
+
+* `<operator-name-prefix>_operaor_health_status` - The metric reports the operator health status. It should be reported by the operator itself based on the internal operator validations.  
```

Comment:
there is a typo in the metrics name: `operaor`

Reply:
Thanks. Updated.